### PR TITLE
feature(EJ2-63880): Getting started for Message control

### PR DESCRIPTION
**Pull request by Deepika5698**
### Feature description

Getting started documentation for New Message control.
 
### Analysis and design

-  Steps for creating the MVC and Core project.
-  Steps to add Message control in MVC and Core application

 
### Solution description

Added Getting started steps for Message control and output screenshot
 
 
### Output screenshots

NA
 
### Areas affected and ensured
No area is affected due to this changes.
 
### Test cases
NA
 
### Test bed sample location
NA
 
### Additional checklist
 
* Did you run the automation against your implementation? NA
* Is there any API name change? No
* Is there any existing behavior change of other features due to this code change? No
* Does your new code introduce new warnings or binding errors? No
* Does your code pass all FxCop and StyleCop rules? Yes
* Did you record this case in the unit test or UI test? NA

### CODE REVIEW

**Review by Mydeensn — APPROVED** [review]

File: ej2-asp-core-mvc/message/EJ2_ASP.MVC/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
+---
+layout: post
+title: Getting Started with ##Platform_Name## Message Control | Syncfusion
+description: Checkout and learn about getting started with ##Platform_Name## Message control of Syncfusion Essential JS 2 and more details.
+platform: ej2-asp-core-mvc
+control: Getting Started
+publishingplatform: ##Platform_Name##
+documentation: ug
+---
+
```
Comment (PiramanayagamSyncfusion) [inline ej2-asp-core-mvc/message/EJ2_ASP.MVC/getting-started.md:10]: remove unwanted empty space

Comment (Deepika5698) [reply ej2-asp-core-mvc/message/EJ2_ASP.MVC/getting-started.md:10]: Removed unwanted spacings.

Comment (Deepika5698) [reply ej2-asp-core-mvc/message/EJ2_ASP.MVC/getting-started.md:10]: Reverted the changes again. If I removed spacing, then file path error exists.

**Comment by Deepika5698** [comment]
rebuild

**Comment by Deepika5698** [comment]
Moved this changes in below PR. So I am closing this PR.
https://github.com/syncfusion-content/ej2-asp-core-mvc-docs/pull/891

**Pull request closed without merging**